### PR TITLE
add axios v0.15

### PR DESCRIPTION
**Pull request by marudor**
add Cancellation & proxy settings

### CODE REVIEW

File: definitions/npm/axios_v0.15.x/flow_v0.28.x-/axios_v0.15.x.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  declare type $AxiosXHRConfig<T> = AxiosXHRConfig<T>;
+  declare class AxiosXHR<T> {
+    config: AxiosXHRConfig<T>;
+    data: T;
+    headers: Object;
+    status: number;
+    statusText: string,
+    request: http$ClientRequest | XMLHttpRequest
+  }
+  declare type $AxiosXHR<T> = $AxiosXHR<T>;
```
Comment (torifat) [inline definitions/npm/axios_v0.15.x/flow_v0.28.x-/axios_v0.15.x.js]: @marudor Is this ok? Or, is it `declare type $AxiosXHR<T> = AxiosXHR<T>;`

Comment (marudor) [inline definitions/npm/axios_v0.15.x/flow_v0.28.x-/axios_v0.15.x.js]: I actually copy pasted that. I think its legacy.. Its not used - nor does it make any sense.
I'm going to fix that later. Thanks for catching.

Comment (torifat) [inline definitions/npm/axios_v0.15.x/flow_v0.28.x-/axios_v0.15.x.js]: There are a few other minor changes like `defaults` & missing `common` in `headers`, etc. I will will send a PR with with everything.

Comment (marudor) [inline definitions/npm/axios_v0.15.x/flow_v0.28.x-/axios_v0.15.x.js]: Even better :)

**Pull request merged**